nginx/Dockerfile

```diff
@@ -53,6 +62,8 @@ ENV \
     PHP_MAX_INPUT_VARS=3000 \
     PHP_MEMORY_LIMIT=256M \
     PHP_POST_MAX_SIZE=128M \
+    PHP_PROCESS_CONTROL_TIMEOUT=60 \
+    PHP_REQUEST_TERMINATE_TIMEOUT=60 \
```

Comment:
The default is `0` according to the documentation above for both of the `php` settings?